src/main/java/uk/gov/justice/digital/hmpps/csr/api/dto/NotificationDto.kt

```diff
@@ -7,14 +7,14 @@ import io.swagger.annotations.ApiModelProperty
 import org.slf4j.LoggerFactory
 import uk.gov.justice.digital.hmpps.csr.api.domain.ActionType
 import uk.gov.justice.digital.hmpps.csr.api.domain.ShiftType
-import uk.gov.justice.digital.hmpps.csr.api.model.ShiftDetail
+import uk.gov.justice.digital.hmpps.csr.api.model.DetailNotification
```

Comment:
Is it worth dropping the term "notification" from `DetailNotification` and `ShiftNotification`? It doesn't feel like it adds any value and makes both a bit more wordy. Come to think of it, I imagine the concept of a "notification" only really comes into play in `cmd-api`, in this service we're just getting modifications.

Reply:
you're right. I think the endpoint shouldn't be /notifications even, its /planunit/{id}/shift?from=&to=&modifiedOnly=true and  /planunit/{id}/detail?from=&to=&modifiedOnly=true 
what do you think?